Doc/library/socket.rst

```diff
@@ -1091,6 +1091,17 @@ The :mod:`socket` module also offers various network-related services:
    .. versionchanged:: 3.8
       Windows support was added.
 
+   .. note::
+
+      On Windows network interfaces have different names in different contexts:
+
+      * UUID: ``{FB605B73-AAC2-49A6-9A2F-25416AEA0573}``
+      * name: ``ethernet_32770``
+      * friendly name: ``vEthernet (nat)``
+      * description: ``Hyper-V Virtual Ethernet Adapter``
+
+      This function returns names of the second form from the list.
```

Comment:
This last sentence is perhaps not as clear as it could be. Perhaps mention the example value `etherhnet_32770` explicitly?

Reply:
That's fair, I'll do that in a minute.